custom-order-numbers-for-woocommerce/custom-order-numbers-for-woocommerce.php

```diff
@@ -26,7 +26,15 @@
 
 if ( is_admin() ) {
 	add_filter( 'plugin_action_links_' . plugin_basename( __FILE__ ), 'action_links' );
-
+	$plugin_file = 'custom-order-numbers-for-woocommerce-pro/custom-order-numbers-for-woocommerce-pro.php';
+	if (
+		in_array( $plugin_file, apply_filters( 'active_plugins', get_option( 'active_plugins', array() ) ), true ) ||
+		( is_multisite() && array_key_exists( $plugin_file, get_site_option( 'active_sitewide_plugins', array() ) ) )
+	) {
+		if ( '1.4.1' !== get_option( 'alg_custom_order_numbers_version', '' ) ) {
```

Comment:
use version_compare function to compare the versions and we shall not check if its not equal to 1.4.1.
If we keep the above condition then the notice will again be displayed for version 1.4.2 onwards.

Suggested change:
```
if ( version_compare( get_option( 'alg_custom_order_numbers_version', '' ), '1.4.1' ) < 0 ) {
```